Documentation/DocuBlocks/Rest/Graph/general_graph_edge_definition_remove_http_examples.md

```diff
@@ -18,7 +18,7 @@ The name of the edge collection used in the definition.
 
 @RESTQUERYPARAMETERS
 
-@RESTQUERYPARAM{dropCollection,boolean,optional}
+@RESTQUERYPARAM{dropCollections,boolean,optional}
```

Comment:
Can we please remove this change from this PR?
Unreleated and contained somewhere else.